.github/workflows/benchmark.yml

```diff
@@ -32,12 +32,22 @@ on:
         required: true
         description: 'Dependency packages, you can also set a specific version'
         type: string
-        default: 'packaging transformers_stream_generator transformers datasets matplotlib'
+        default: 'packaging transformers_stream_generator transformers datasets matplotlib jmespath'
```

Comment:
what's `jmespath` used for?

Reply:
> what's `jmespath` used for?

by using torch >= 2.2.1 need this packages. just keep for future torch version update.

Reply:
Odd. What is it? Which model brings it in?

Reply:
> Odd. What is it? Which model brings it in?

I check test history . I add it because of this error. It seems only affect profile_generation script. 

```
python3 benchmark/profile_generation.py /nvme/qa_test_models/internlm/internlm2-chat-20b --tp 2 --backend pytorch -c 128 -ct 128 2048 128 -pt 128 128 2048
profiling ... concurrency: 128, n_prompt_token: 128, n_completion_token: 128, test_round: 3, warmup_round: 1
Traceback (most recent call last):
  File "/home/zhulin1/lmdeployMainNew/benchmark/profile_generation.py", line 472, in <module>
    main()
  File "/home/zhulin1/lmdeployMainNew/benchmark/profile_generation.py", line 425, in main
    output = _process_map(profile_target, (args.model_path, ))
  File "/home/zhulin1/lmdeployMainNew/benchmark/profile_generation.py", line 372, in _process_map
    raise ret
ModuleNotFoundError: No module named 'jmespath'
```

Reply:
I think it not related to torch 2.2.1 version. It's related to latest main code?